packages/components/src/filters/date/style.scss

```diff
@@ -91,11 +91,6 @@ button.woocommerce-filters-date__tab {
 	&.is-custom {
 		border-top: 1px solid $core-grey-light-700;
 	}
-
-	&.is-sticky-bottom {
-		position: absolute;
-		bottom: 0;
-	}
```

Comment:
Looking on Chrome emulation mode, I see the bottom cut off:

<img width="355" alt="Screen Shot 2019-04-05 at 3 44 27 PM" src="https://user-images.githubusercontent.com/1922453/55600737-dd026a00-57b9-11e9-8472-f8f5d408bf0f.png">

This was the issue I was trying to prevent with `is-sticky-bottom`. I do think your way is better though, its just that the contents aren't scrollable (at least on Chrome)

Reply:
Good catch with that @psealock!

> Looking on Chrome emulation mode, I see the bottom cut off:

I imagine we have to make a trade-off for small screens: or we show the bottom cut off or overlapping the calendar.

I made the datepicker scrollable in 6cdd74df3ccfc0ff55fc5e80704bfaa7d5ed4179, I think that's a good enough solution, but let me know what you think.